scss/utilities/_inputs.scss

```diff
@@ -182,3 +182,7 @@ input:disabled ~ label,
 textarea:disabled ~ .form-text {
     font-style: italic;
 }
+
+input[readonly], textarea[readonly] {
```

Comment:
```suggestion
input:read-only,
textarea:read-only {
```

For this to pass the stylelint check, different selectors have to be on separate lines. Also adjusted this to use the `:read-only` pseudo selector.